### ISSUE TITLE
Following README.md => unexpected token export

### ISSUE THREAD

**Issue by jcjollant** [cazala/synaptic#256]
I am trying to use this lib with nodeJS v6.10.3 on Windows
I followed the readme (npm+example) which gives the following:
```
C:\test\node_modules\synaptic\src\synaptic.js:1
(function (exports, require, module, __filename, __dirname) { export {default as
 Neuron} from './Neuron';
                                                              ^^^^^^
SyntaxError: Unexpected token export
    at createScript (vm.js:56:10)
    ...
```
The code is not too ambitious:
`var synaptic = require('synaptic');
`

Unless I am missing something obvious, instructions do not work.

**Comment by fredylg** [comment]
try this var synaptic = require(__dirname + '/dist/synaptic.js');

**Comment by Loksly** [comment]
another solution is just to install via:

    npm install synaptic

like you already did, then the edit the node_modules/synaptic/package.json file, line 73, change the line

    "main": "./src/synaptic",

to:

    "main": "./dist/synaptic",

**Comment by jcjollant** [comment]
@fredylg __dirname was undefined, but it worked pointing directly at the dist synaptic.js . Thank you
@Loksly It also worked with your suggestion in package.json . Thank you


I still feel README.md should be fixed to mention these tricks.

**Comment by cazala** [comment]
Hi @jcjollant, this was due to the `v1.1.0` release from 2 days ago. I just released a hotfix `v1.1.1` where this is fixed. You can just bump the version in your dependencies to 1.1.1 and reinstall synaptic, it should be working
